modules/bucksenseBidAdapter.js

```diff
@@ -0,0 +1,171 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER } from 'src/mediaTypes';
+
+const WHOIS = 'BKSHBID-008';
+const BIDDER_CODE = 'bucksense';
+const URL = 'https://prebid.bksn.se:445/prebid';
```

Comment:
Is this the correct endpoint URL for your adapter?

When I tried to test the adapter, the POST requests to this URL were resulting in a 301 Moved Permanently response.

Reply:
Yes request is doing a 301 but then got redirect to the script that return the BID. I have tested the prebid on the Hello World example. Let me know if you do not see the response at all from your test.